content/en/docs/collector/transforming-telemetry.md

```diff
@@ -0,0 +1,176 @@
+---
+title: Transforming telemetry
+weight: 31
```

Comment:
Change the weight so that it appears below Scaling, and before all the "Build" pages. (Right now it's in the middle of the "Build" pages -- see below.)

```suggestion
weight: 26
```

We'll have to space those weights out in a followup PR. /cc @svrnm @cartermp 

> <img width="285" alt="image" src="https://user-images.githubusercontent.com/4140793/218588654-e3f65271-8981-4510-8bfd-9228c920d602.png">